libslide/src/bignum/mul.rs

```diff
@@ -28,7 +28,7 @@ impl ops::Mul for Bignum {
             rhs_int.push(0);
             lhs_int.push(0);
         }
-        while !ispowerof2(rhs_int.len() + rhs_dec.len()) {
+        while !ispowerof2(rhs_int.len() + rhs_dec.len()) || rhs_int.len() <= 2 {
```

Comment:
Why not add the `rhs_dec.len() <= 2` condition as well?

Reply:
It should the same. This is a mistake.